frontend/src/components/Select.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components/macro'
+import Label from './Label'
+
+export default function Select({
+  name,
+  selected,
+  title,
+  onChange,
+  value,
+  values,
+  ...props
+}) {
+  return (
+    <Label {...props}>
+      {title}
+      <SelectStyled name={name} value={value} onChange={onChange}>
+        <option value="">bitte wählen...</option>
+        {values.map(value => (
+          <option key={value} value={value}>
+            {value}
+          </option>
```

Comment:
vielleicht in eine function oberhalb des returns auslagern?

Reply:
Guter Punkt fürs Refactoring